monkestation/code/datums/votes/transfer_vote.dm

```diff
@@ -46,5 +53,17 @@
 	if(isobserver(voter) || voter.stat == DEAD || !(voter.ckey in GLOB.joined_player_list)) // only living crew gets to vote
 		return FALSE
 
+/datum/vote/shuttle_call/tiebreaker(list/winners)
+	return CHOICE_CONTINUE // Generally, this is less likely to make people mad. It still can, don't get me wrong, but it's safer.
+
+/datum/vote/shuttle_call/get_vote_result(list/non_voters)
+	for(var/ckey in non_voters)
+		var/client/client = non_voters[ckey]
+		if(client.mob && can_vote(client.mob))
```

Comment:
just to be safe
```suggestion
		if(client?.mob && can_vote(client.mob))
```

Reply:
This is not required due to an explicit handling of nulls in previous code, but I'll do it anyway.

Reply:
always assume a client can *somehow* be null, even if you really really think they can't be.